src/assets/i18n/en.json5

```diff
@@ -1251,6 +1252,14 @@
 
   "file-section.error.header": "Error obtaining files for this item",
 
+  
+  "idle.modal.header":"Your Session is About to Expire!",
+
+  "idle.modal.info":"Your session is about to expire, would you like to logout or continue the session?",
+
+  "idle.modal.log-out":"Logout",
+
+  "idle.modal.extend-session":"Continue Session",
```

Comment:
@josekarvalho : These 4 already exist in the latest `main` code.  They are at the very end of the file. See: https://github.com/DSpace/dspace-angular/blob/main/src/assets/i18n/en.json5#L3733-L3739

So, these can be removed from your PR

Reply:
ok, I searched on another place (https://github.com/DSpace/dspace-angular/blob/main/src/assets/i18n/en.json5)

Reply:
I was checking to update the pull request but... the strings are different.
In this case, you have "idle-modal..." and I suggest "idle.modal..."

It may be necessary an update on the existing expressions, or both strings are used...

Reply:
@josekarvalho : I just searched the dspace-angular codebase, and I don't see any usages of "idle.modal.*". I'm only seeing "idle-modal.*"

So, I'm confused why "idle.modal" is needed?

Reply:
@tdonohue , ok, it may be some error on putting the string to the file. I will remove these.